src/triage/cli.py

```diff
@@ -177,7 +177,18 @@ def __init__(self, parser):
             "--n-processes",
             type=natural_number,
             default=1,
-            help="number of cores to use",
+            help="number of cores to use for small classifiers (e.g. Logistic Regression)",
+        )
+        parser.add_argument(
+            "--n-bigtrain-processes",
+            type=natural_number,
+            default=1,
+            help="number of cores to use for big, computationally-intensive classifiers (e.g. Random Forests)",
+        )
+        parser.add_argument(
+            "--additional-bigtrain-classnames",
+            nargs="*",
+            help="Additional classifier names to train alongside the 'big' classifiers like random forests.",
```

Comment:
Do you specify multiple by repeating this argument or providing a list after the argument (if so, how should the list be delimited?)

Reply:
Repeating the argument

Reply:
Makes sense -- would it be clearer to call the CLI argument `--additional-bigtrain-classname` in that case, or does making it plural give a hint that it can be repeated (I'm not sure of the best/standard practice there)?

Reply:
I think singular is good. It also seems that with repeatable options that short option names are better. Heavily abbreviated ones, even. It feels weird to do something like 'abc' (the actual abbreviation here):

`triage --abc classname1 --abc classname2`

But that actually seems kind of common? On the other hand this seems like a weird time to introduce it, given that we don't have abbreviated names for other options. If we start adding abbreviated option names we should probably do it in one go.

Reply:
But maybe slightly shorter?  Is "additional" extraneous here, or could be shortened to addtl or add?

`add_bigtrain_class`? That seems like it could be just as readable but shorter to type

Reply:
Actually, plural is better, and you'll read why when I upload the docs change